Assoumpta/MyDiary
# Ft acc 168055167

What does this PR do?
This PR shows login and signup forms where a User will be able to fill it by creating account or login.

Description of Task to be completed?
Design signup form for the new users
Design login for the users who have already account.


How should this be manually tested?
clone a repository.
git checkout ft-acc-168055167

Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
Create a repository.
As a user I should be able to create an account and Sign in with my account.
#### Screenshots (if appropriate)
![signup](https://user-images.githubusercontent.com/54316466/64230657-46d68b00-cea2-11e9-8a1c-826c52b3b670.PNG)
![login](https://user-images.githubusercontent.com/54316466/64230658-46d68b00-cea2-11e9-8e37-bd80029f1e6b.PNG)

#### Questions:
N/A
